tuning/utils/preprocessing_utils.py

```diff
@@ -25,25 +25,141 @@
 from tuning.config import configs
 
 
-def validate_data_args(
+def validate_data_args(data_args: configs.DataArguments, packing: bool):
+
+    assert isinstance(
+        data_args.training_data_path, str
+    ), "Training data path has to be set and str"
+
+    # Dataset containing single sequence needs a response template for masking
+    if data_args.response_template is None and data_args.dataset_text_field is not None:
```

Comment:
Isn't this condition fully subsumed by the next error condition? The next error condition is when packing is false and response_template is empty. This one is the same condition AND dataset_text_field is non-empty. The error will be spotted even without this condition - is the intent to provide a specific error message?